app/assets/javascripts/components/common/date_picker.jsx

```diff
@@ -25,7 +26,8 @@ const DatePicker = React.createClass({
     onFocus: React.PropTypes.func,
     onClick: React.PropTypes.func,
     append: React.PropTypes.string,
-    date_props: React.PropTypes.object
+    date_props: React.PropTypes.object,
```

Comment:
Along with datepicker unit tests, it would be great to use `React.PropTypes.shape` to validate the shape of date_props more precisely. Not strictly necessary as part of this PR, but it would be nice.